.eslintrc.js

```diff
@@ -49,6 +49,7 @@ module.exports = {
   ignorePatterns: [
     '/libs/deps/*',
     '/tools/loc/*',
+    '/libs/features/spectrum-web-components/*',
```

Comment:
without this the run-lit github workflow fails to run eslint on changed files since it cannot find the npm dependencies. 
Since those are generated as bundles in the dist folder, this is indeed not necessary.